pocketbook/cli.py

```diff
@@ -173,6 +176,56 @@ def run_add(args):
     address_book = AddressBook()
     address_book.add(args.name, args.address)
 
+def run_import(args):
+    key_store = KeyStore()
+
+    # get the name for the new key
+    while True:
+        name = input('Enter name for key: ')
+        if name in key_store.list_keys():
+            print('Key name already exists')
+            continue
+        break
+
+    # prompt the user for the password
+    while True:
+        key_str = getpass.getpass('Enter private key (hex of b64 encoded)...: ')
+
+        try:
+            e = Entity.from_hex(key_str)
```

Comment:
Since we know the exact length of the key string I think it would be cleaner to do a regex match against the desired format first beforehand, thoughts?